sites/docs/src/content/docs/usage/Getting_started/configuration.md

```diff
@@ -112,18 +112,42 @@ For more information on how to specify an executor, please refer to the [Nextflo
 
 In addition to the executor, you may find that pipeline runs occasionally fail due to a particular step of the pipeline requesting more resources than you have on your system.
 
+To avoid these failures, you can tell Nextflow to set a cap pipeline-step resource requests against a list called `resourceLimits` specified in Nextflow config file. These should represent the maximum possible resources of a machine or node.
+
+For example, you can place into a file the following:
+
+```groovy
+process {
+  resourceLimits = [
+    cpus: 32,
+    memory: 256.GB,
+    time: 24.h
+  ]
+}
+```
+
+And supply this in your pipeline run command with `-c <custom>.config`. Then, during a pipeline run, if for example a job exceeds the default memory request, it will be retried increasing the memory each time until either the job completes, or until it reaches a request of `256.GB`.
+
+Therefore, these parameters only act as a _cap_, to prevent Nextflow submitting a single job requesting resources more than what is possible on your system and requests getting out of hand.
+
+Note that specifying these will not _increase_ the resources available to the pipeline tasks! See [Tuning workflow resources](#tuning-workflow-resources) for this.
+
+:::note{collapse title="Note on older nf-core pipelines"}
+
+If you wish to use pipelines generated with the pre-`v3.0.0` nf-core template, and/or when running with Nextflow versions earlier than 24.04.0 you may need use a different syntax to prevent the resources from exceeding a maximum limit.
```

Comment:
```suggestion
If you wish to use pipelines generated with the nf-core template before `v3.0.0`, and/or when running with Nextflow versions earlier than 24.04.0 you may need use a different syntax to prevent the resources from exceeding a maximum limit.
```